examples/expression-oriented-style/expression_oriented_style.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/io;
+
+// You can define the return value expression using the `=>` notation instead of using
+// curly braces to define the function body block when the function body is just
+// a return statement with an expression.
```

Comment:
```suggestion
// You can define the body of a function using the `=>` notation followed by the return value expression 
// instead of using curly braces to define the function body block when the function body is just
// a return statement with an expression.
```